src/MongoDB/BulkWrite.c

```diff
@@ -374,7 +371,7 @@ static PHP_METHOD(BulkWrite, insert)
 
 	intern->num_ops++;
 
-	if (bson_out && return_value_used) {
+	if (bson_out) {
```

Comment:
I wonder if this condition pre-dates the `cleanup` label below. Would `bson_out` only still be null if `php_phongo_zval_to_bson` failed? And in that case, would we have already jumped to `cleanup`? Either way, I'm OK leaving this as a sanity check.

Interestingly, the `cleanup` label was added in https://github.com/mongodb/mongo-php-driver/commit/1e0b1e8e290699101c21b20e324a9b154df389cd, which is separate from the `goto cleanup;` lines added in https://github.com/mongodb/mongo-php-driver/commit/7ff55ba42cb36f60836d2d83426fa909c5b54a8e (same release, though). Perhaps I just goofed in splitting up those commits.

Reply:
`bson_out` will always be set due to us using `PHONGO_BSON_RETURN_ID` a few lines earlier, so the check could definitely go away, but would most likely cause a segmentation fault in `php_phongo_bulkwrite_extract_id` if it wasn't set for some reason.

While a sanity check makes sense, it should most likely be an error condition and not silently be a no-op. I've updated this accordingly.